client/dashboard/app/router.tsx

```diff
@@ -278,6 +282,27 @@ const siteSettingsDefensiveModeRoute = createRoute( {
 	)
 );
 
+const siteSettingsSftpSshRoute = createRoute( {
+	getParentRoute: () => siteRoute,
+	path: 'settings/sftp-ssh',
+	loader: async ( { params: { siteSlug } } ) => {
+		const site = await queryClient.ensureQueryData( siteQuery( siteSlug ) );
+		if ( canUseSftp( site ) ) {
+			await queryClient.ensureQueryData( siteSftpUsersQuery( siteSlug ) );
+		}
+
+		if ( canUseSsh( site ) ) {
+			await queryClient.ensureQueryData( siteSshAccessStatusQuery( siteSlug ) );
+		}
```

Comment:
`Promise.all()` doesn't mind if you pass it a boolean (it's as if its passed an already resolved boolean promise), so you can still run the queries in parallel like this.

```suggestion
		return Promise.all( [
			canUseSftp( site ) && queryClient.ensureQueryData( siteSftpUsersQuery( siteSlug ) ),
			canUseSsh( site ) && queryClient.ensureQueryData( siteSshAccessStatusQuery( siteSlug ) ),
		] );
```

Reply:
Done by https://github.com/Automattic/wp-calypso/pull/103783/commits/276ce4c86d39fdfd62d225b697bdcf0b435dbf39.